### ISSUE TITLE
Error running msf host builder

### ISSUE THREAD

**Issue by usmcfiredog** [rapid7/metasploit-baseline-builder#35]
python build_msf_host.py 
  0%|                                                     | 0/1 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "build_msf_host.py", line 106, in <module>
    main(sys.argv)
  File "build_msf_host.py", line 101, in main
    build_base(target, replace_existing=replace_vms, vmServer=vm_server, prependString=prependString)
  File "build_msf_host.py", line 59, in build_base
    p.build(parallel=True, debug=False, force=False)
  File "/Users/nicholaskasprinski/.pyenv/versions/2.7.13/lib/python2.7/site-packages/packer.py", line 63, in build
    return self.packer_cmd()
  File "/Users/nicholaskasprinski/.pyenv/versions/2.7.13/lib/python2.7/site-packages/sh.py", line 1021, in __call__
    return RunningCommand(cmd, call_args, stdin, stdout, stderr)
  File "/Users/nicholaskasprinski/.pyenv/versions/2.7.13/lib/python2.7/site-packages/sh.py", line 486, in __init__
    self.wait()
  File "/Users/nicholaskasprinski/.pyenv/versions/2.7.13/lib/python2.7/site-packages/sh.py", line 500, in wait
    self.handle_command_exit_code(exit_code)
  File "/Users/nicholaskasprinski/.pyenv/versions/2.7.13/lib/python2.7/site-packages/sh.py", line 516, in handle_command_exit_code
    raise exc(self.ran, self.process.stdout, self.process.stderr)
sh.ErrorReturnCode_1: 

  RAN: '/usr/local/bin/packer build -parallel=true -only=vmware-iso -var vm_name=msf_host -var iso_url=http://releases.ubuntu.com/16.04/ubuntu-16.04.4-server-amd64.iso -var disk_size=65536 -var iso_checksum=ee834fbeb94cc55972b38caafa2029c29625e2e8 -var preseed=preseed.cfg -var _comment=Build with `python build_msf_host.py` -var iso_name=ubuntu-16.04.4-server-amd64.iso -var cpus=2 -var desktop=true -var custom_script=../../scripts/msf_host/msf_host_setup.sh -var ssh_password=vagrant -var memory=1024 -var ssh_fullname=msfuser -var output=../../box/msf_host_vmware.box -var ssh_username=msfuser -var iso_checksum_type=sha1 ../../tmp/msf_host/msf_host.json'

  STDOUT:

  STDERR:

One issue that I thought it was, is that ubuntu 16.04.4 is no longer available at the link in the json file but even after that I am still getting the above error.

**Comment by usmcfiredog** [comment]
yes, I did.

When I get home today I will do it again to make sure.

**Comment by usmcfiredog** [comment]
Sorry hit the wrong button this issue is not closed

**Comment by usmcfiredog** [comment]
Stupid question where would I go to make sure they downloaded after running those commands? I know I have ran them in the directory where I have the msf repository downloaded

**Comment by usmcfiredog** [comment]
I just made sure and ran those commands and it is still giving me that error.

**Comment by usmcfiredog** [comment]
Ok weird thing so I ran those commands again in the original directory and nothing happened and got the same error. I then decided to create a new directory and clone the repo and ran those commands. It failed the first time with same error and then it worked when I ran the python script again. Sorry don't have any more or other info.

**Comment by usmcfiredog** [comment]
Ok so now it is starting to build but then fails and gives the same error as above. I have also attached the output file from the build.

[output.log](https://github.com/rapid7/metasploit-baseline-builder/files/2798630/output.log)

**Comment by jmartin-r7** [comment]
Specific issue here was fixed by #34

```
[0;32m    vmware-iso: Downloading https://github.com/rvm/rvm/archive/1.29.7.tar.gz[0m
[0;32m    vmware-iso: Downloading https://github.com/rvm/rvm/releases/download/1.29.7/1.29.7.tar.gz.asc[0m
[0;32m    vmware-iso: gpg: Signature made Thu 03 Jan 2019 10:01:48 PM UTC using RSA key ID 39499BDB[0m
[0;32m    vmware-iso: gpg: Can't check signature: No public key[0m
[0;32m    vmware-iso: GPG signature verification failed for '/home/msfuser/.rvm/archives/rvm-1.29.7.tgz' - 'https://github.com/rvm/rvm/releases/download/1.29.7/1.29.7.tar.gz.asc'! Try to install GPG v2 and then fetch the public key:[0m
[0;32m    vmware-iso:[0m
[0;32m    vmware-iso:     sudo gpg2 --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB[0m
[0;32m    vmware-iso:[0m
[0;32m    vmware-iso: or if it fails:[0m
[0;32m    vmware-iso:[0m
```
